### ISSUE TITLE
🔨 Make baby Ul'xzomit follow mother Ulxzomit

### ISSUE THREAD

**Issue by DiscipleOfEris** [DerpyProjectGroup/topaz#2604]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)

## Describe the feature

In Al'Taieu, each Ul'xzomit is supposed to be followed by two smaller Ul'xzomit. The mother and baby Ul'xzomit also have different drop rates.

See:
 * https://www.youtube.com/watch?v=QVLQUynPjvs
 * https://ffxiclopedia.fandom.com/wiki/Ul%27xzomit

The babies are supposed to spawn after the mother, and they will link with the mother but not vice versa (nor with eachother). The wiki page suggests there are 45 mothers and 90 babies, but the SQL tables have only 45 Ul'xzomit entries, so it's not clear what the mob ids of the babies would be. Considering the Ul'xzomit ids always appear in groups of three, I suggest having 15 mother Ul'xzomit and making the two subsequent Ul'xzomit in the table be babies.

**Comment by TeoTwawki** [comment]
the client dat only has 45 entries. The wiki's tend to list total spawn locations rather than total mobs that could be up at once, and it is done by player report rather than confirmed in the data files. I can tell you every group of 3 in teh dat is one momma and 2 babies, and there are no further mobs because that's what is in the dat in groups of 3 repeated:

```
16912394	Ul'xzomit
16912395	Ul'xzomit
16912396	Ul'xzomit
< other mobs >
16912409	Ul'xzomit
16912410	Ul'xzomit
16912411	Ul'xzomit
< other mobs>
16912422	Ul'xzomit
16912423	Ul'xzomit
16912424	Ul'xzomit
```

tl;dr you are exactly right and wiki is wrong :P `#shitWikiSays`
